test/configs/latest_storage_block_size_16kB.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "Run everything with the latest storage on databases with a default block size of 16kB.",
+  "on_init": "ATTACH '__TEST_DIR__/{BASE_TEST_NAME}block_size.db' AS __test__config__latest_storage_block_size_16kB (BLOCK_SIZE 16384, STORAGE_VERSION 'latest'); SET storage_compatibility_version='latest';",
```

Comment:
`"ATTACH '__TEST_DIR__/{BASE_TEST_NAME}block_size.db'`

Should we give the files themselves a very long identifier, to also avoid file conflicts? 😅